### PR TITLE
Quick fix milesight

### PR DESCRIPTION
**Pull request by J2thatsme**
Milesight uplinks & added schema for the Device simulator & fixed yanzi uplink

### CODE REVIEW

File: types/internal/deviceSimulator/default.schema.json [hunk 1]
```diff
@@ -17,13 +12,40 @@
...
+      "minimum": 20,
+      "maximum": 120
+    },
+    "illuminance": {
+      "title": "Light",
+      "type": "number",
+      "unit": "lux",
+      "description": "Light value"
+    },
+    "occupancy": {
```
Comment (mariusgiger) [inline types/internal/deviceSimulator/default.schema.json:45]: why move away from occupied?

File: types/milesight/WS202/uplink.js [hunk 2]
```diff
@@ -38,5 +38,5 @@ function consume(event) {
   }
 
   emit("sample", { data: lifecycle, topic: "lifecycle" });
-  emit("sample", { data, topic: "button_pressed" });
+  emit("sample", { data, topic: "defaul" });
```
Comment (mariusgiger) [inline types/milesight/WS202/uplink.js:41]: typo default

**Review by mariusgiger — COMMENTED** [review]
Looks good, two small adjustments

File: types/internal/deviceSimulator/default.schema.json (hunk 1 above)
Comment (J2thatsme) [reply types/internal/deviceSimulator/default.schema.json:45]: Occupancy seems to be used more often than occupied.

Comment (mariusgiger) [reply types/internal/deviceSimulator/default.schema.json:45]: occupied makes more sense when it comes to the name or not?

Comment (mariusgiger) [reply types/internal/deviceSimulator/default.schema.json:45]: I would differentiate between state which can take more values than just true/false and purely boolean values

**Review by mariusgiger — APPROVED** [review]

**Pull request merged**